.github/workflows/linux.yml

```diff
@@ -11,8 +11,8 @@ jobs:
   build:
     runs-on: ubuntu-20.04
     steps:
+    - name: Install Dependencies
+      run: sudo apt update && sudo apt install -y cmake
     - uses: actions/checkout@v3
-
     - name: Build
-      # TODO: Split these into separate lines.
-      run: sudo apt update && sudo apt install -y cmake && cmake -S . -B build && cmake --build build
+      run: cmake -S . -B build && cmake --build build
```

Comment:
openssl needed only for mac and windows?

Reply:
For this one I cheated a bit. I picked a version of Ubuntu that comes pre-packed with OpenSSL 1.1.

I will add a new task to script the installation of OpenSSL so we can get to a more modern version (or test on multiple versions of Ubuntu)